pkg/gather/clusterconfig/operators.go

```diff
@@ -61,7 +61,7 @@ type CompactedEventList struct {
 }
 
 // GatherClusterOperators collects all ClusterOperators and their resources.
-// It finds unhealthy Pods for unhealthy operators
+// It finds Pods for unhealthy operators
```

Comment:
This doesn't tell much IMO. It would be nice to describe it more.

Reply:
Check it now.